site/en/blog/lighthouse-8-4/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 8.4
+authors:
+  - brendankenny
+date: 2021-09-09
+description: >
+  Lighthouse 8.4 now guards against lazily-loading LCP images and failing FID without a mobile viewport.
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  # - chrome-95
+---
+
+Lighthouse 8.4 is available immediately on the command line (`npm install -g lighthouse`) and in Chrome Canary, expected to ship in Chrome 95, and to be available in [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) within two weeks.
+
+See the full list of changes in the [8.4 changelog](https://github.com/GoogleChrome/lighthouse/releases/tag/v8.4.0).
+
+## New Audits
+
+### Don't lazy load Largest Contentful Paint images {: #new-audit-lazy-lcp }
+
+Lazy loading images can be an effective way to defer offscreen images so they won't interfere with quickly loading content that is on screen.
+
+However, if a page's [LCP](https://web.dev/lcp/) element is an image, lazy loading it can have a significant negative effect on the LCP. The browser may queue loading of the image and fetch other resources first, instead of prioritizing the image for immediate download. A [recent study of lazy loading in WordPress](https://web.dev/lcp-lazy-loading/) found that LCP can improve by as much as 15% for some sites if images in the initial viewport are not lazy loaded.
+
+{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="The lazy-loaded LCP audit in a Lighthouse report", width="800", height="502", style="border: 1px solid var(--color-hairline)" %}
```

Comment:
> `style="border: 1px solid var(--color-hairline)"`

there was a handy `.w-screenshot` class for this on web.dev. Happy to do this, though, if it's not uncouth to use an inline style. Should it be switched to `s/var(--color-hairline)/#dadce0` so it's a little more robust to future site style changes (at the expense of not catching any future changes to `--color-hairline`)?

Reply:
I think the class is just called `.screenshot` on dcc. Did you try that?

Reply:
https://developer.chrome.com/docs/handbook/components/#images

Reply:
ha, I swear I looked through the css but apparently missed that. Thanks.